src/Shortcode/Shortcode.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace BusinessProfileRender\ShortCode;
+
+/**
+ * Class AdminNotice
+ */
+class ShortCode
+{
+    public static function init()
+    {
+        add_shortcode("business_profile", [
+            __CLASS__,
+            "render_business_profile",
+        ]);
+    }
+
+    // Method to check if bpr_business_profile option exists
+
+    public static function render_business_profile($attr)
+    {
+        // Parse JSON from wp_options
+        $json_data = get_option("bpr_business_profile");
```

Comment:
Add `bpr_business_profile` as defined variable 

`define( 'BUSINESS_PROFILE_RENDER_OPTION', 'bpr_business_profile' );`

Reply:
Var defined in business-profile-render.php file